DependencyInjection/Configuration.php

```diff
@@ -15,8 +17,13 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('yokai_enum');
+        if (version_compare(Kernel::VERSION, '4.2') >= 0) {
+            $treeBuilder = new TreeBuilder('presta_sitemap');
```

Comment:
Whoops, you forgot to edit after copy/paste

Reply:
nice catch, thank you